vagrant/provisioning/roles/activemq/tasks/main.yml

```diff
@@ -170,6 +170,34 @@
         dest: "{{ root_folder }}/app/activemq/lib/mariadb-java-client-{{ mariadb_jdbc_version }}.jar"
         state: link
 
+- name: enable authentication for ActiveMQ in activemq.xml
+  become: yes
+  become_user: activemq
+  blockinfile:
+    path: "{{ root_folder }}/app/activemq/conf/activemq.xml"
+    insertafter: "        </persistenceAdapter>"
+    marker: "# {mark} ANSIBLE MANAGED BLOCK: Authentication"
+    block: |
+      <plugins>    
+          <!-- Configure authentication; Username, passwords and groups -->
+          <simpleAuthenticationPlugin>
+              <users>
+                  <authenticationUser username="${activemq.username}" password="${activemq.password}" groups="admins"/>
+                  <authenticationUser username="${arkcase.user}" password="${arkcase.password}" groups="users"/>
+              </users>
+          </simpleAuthenticationPlugin>
+      </plugins>
+
+- name: put authentication credentials in credentials.properties file
+  become: yes
+  become_user: activemq
+  blockinfile:
+    path: "{{ root_folder }}/app/activemq/conf/credentials.properties"
+    insertafter: "# Defines credentials that will be used by components (like web console) to access the broker"
+    marker: "# {mark} ANSIBLE MANAGED BLOCK: Authentication"
+    block: |
+      arkcase.user={{ activemq_user | default('guest', true) }}
```

Comment:
Do we not need the activemq.username and activemq.password entries here?  Why do we need the user id and password in credentials.properties, and also in activemq.xml?